libsolidity/analysis/SyntaxChecker.cpp

```diff
@@ -188,23 +192,23 @@ bool SyntaxChecker::visit(Continue const& _continueStatement)
 {
 	if (m_inLoopDepth <= 0)
 		// we're not in a for/while loop, report syntax error
-		m_errorReporter.syntaxError(_continueStatement.location(), "\"continue\" has to be in a \"for\" or \"while\" loop.");
+		m_errorReporter.syntaxError(_continueStatement.location(), quote("continue", false) + " has to be in a" + quote("for") + "or" + quote("while") + "loop.");
 	return true;
 }
 
 bool SyntaxChecker::visit(Break const& _breakStatement)
 {
 	if (m_inLoopDepth <= 0)
 		// we're not in a for/while loop, report syntax error
-		m_errorReporter.syntaxError(_breakStatement.location(), "\"break\" has to be in a \"for\" or \"while\" loop.");
+		m_errorReporter.syntaxError(_breakStatement.location(), quote("break", false) + " has to be in a" + quote("for") + "or" + quote("while") + "loop.");
```

Comment:
Is this really more readable?

Reply:
because of the `, false`? or generally? Note that it's not only about readability, but also that we can easily change the quotes used depending on the output, e.g. fancy terminals might support quotes like `»this«`